src/dsp_tools/utils/project_create.py

```diff
@@ -35,51 +39,51 @@ def _create_project_on_server(
-        project_remote: Project = try_network_action(
-            action=lambda: project_local.read(),
-            failure_msg=""
-        )
-        print(f"\tWARNING: Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP "
-              f"server. Updating it...")
-        success = False
+        project_remote: Project = try_network_action(lambda: project_local.read())
+        print(f"\tWARNING: Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP server. Updating it...")
+        logger.warning(f"Project '{project_remote.shortname}' ({project_remote.shortcode}) already exists on the DSP server. Updating it...")
         # try to update the basic info
         project_remote, _ = _update_basic_info_of_project(project=project_remote, project_definition=project_definition, verbose=verbose)
-        # It doesn't matter if the update is successful or not: continue anyway, because success is already false. 
+        # It doesn't matter if the update is successful or not: continue anyway, because success is anyways false. 
         # There are other things from this file that can be created on the server, e.g. the groups and users, so the process must continue.
+        return project_remote, False
     except BaseError:
-        success = True
-        project_local = Project(
-            con=con,
-            shortcode=project_definition["project"]["shortcode"],
-            shortname=project_definition["project"]["shortname"],
-            longname=project_definition["project"]["longname"],
-            description=LangString(project_definition["project"].get("descriptions")),
-            keywords=set(project_definition["project"].get("keywords")),
-            selfjoin=False,
-            status=True
-        )
-        project_remote = try_network_action(
-            action=lambda: project_local.create(),
-            failure_msg=f"ERROR: Cannot create project '{project_definition['project']['shortname']}' "
-                        f"({project_definition['project']['shortcode']}) on DSP server."
-        )
-        print(f"\tCreated project '{project_remote.shortname}' ({project_remote.shortcode}).")
-
+        pass
+    
+    success = True
+    project_local = Project(
+        con=con,
+        shortcode=project_definition["project"]["shortcode"],
+        shortname=project_definition["project"]["shortname"],
+        longname=project_definition["project"]["longname"],
+        description=LangString(project_definition["project"].get("descriptions")),
+        keywords=set(project_definition["project"].get("keywords")),
+        selfjoin=False,
+        status=True
+    )
+    try:
+        project_remote = try_network_action(lambda: project_local.create())
+    except BaseError:
+        err_msg = f"Cannot create project '{project_definition['project']['shortname']}' " \
+                  f"({project_definition['project']['shortcode']}) on DSP server."
```

Comment:
I would personally put things like `{project_definition['project']['shortname']}` in a separate variable. It adds some extra lines but I think the code would still be more readable.